src/ansys/fluent/core/__init__.py

```diff
@@ -73,15 +73,16 @@ def version_info() -> str:
     return _VERSION_INFO if _VERSION_INFO is not None else __version__
 
 
-# Setup data directory
-USER_DATA_PATH = platformdirs.user_data_dir(
-    appname="ansys_fluent_core", appauthor="Ansys"
```

Comment:
We need `USER_DATA_PATH` to avoid file modification issue while using file transfer service. We can not remove it or replace it with `os.getcwd()` or `EXAMPLES_PATH`.

Reply:
Reinstated the original logic around file_transfer_service, thanks for raising it.